### PR TITLE
Task homepage

### PR DESCRIPTION
**Pull request by trangkarol**

### CODE REVIEW

**Comment by trangkarol** [comment]
screenshot from 2017-10-23 09-13-57

**Comment by trangkarol** [comment]
ready

File: app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <% if logged_in? %>
+          <li><%= link_to current_user.name, "#", class: "smooth-scroll" %></li>
+        <% else %>
+          <li><%= link_to link_to t("user.login"), login_path, class: "smooth-scroll" %></li>
+          <li><%= link_to t("user.sign_up"), signup_path, class: "smooth-scroll" %></li>
+        <% end %>
+      </ul>
+    </div>
+  </div>
+</nav>
```
Comment (dothidiemthao) [inline app/views/layouts/_navbar.html.erb]: end line

File: app/views/layouts/_rails_default.html.erb [hunk 2]
```diff
@@ -2,4 +2,4 @@
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <%= csrf_meta_tags %>
 <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
-<%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
+<%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
```
Comment (dothidiemthao) [inline app/views/layouts/_rails_default.html.erb]: end line

File: app/views/shared/_documents.html.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <div class="col-md-6">
+      <h3><%= document.name %></h3>
+      <br />
+      <p class="text-justify"><%= document.description %></p>
+      <br />
+      <button class="button-more-success btn" type="button" style="margin:10px;"><%= t "read_more" %></button>
+      <br />
+    </div>
+  </div>
+</div>
```
Comment (dothidiemthao) [inline app/views/shared/_documents.html.erb]: end line

File: app/views/home_pages/_list_documents.html.erb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+              <p class="text-justify"><%= document.description %></p>
+              <br />
+              <%= link_to t("read_more"), "#", class: "button-more-success btn" %>
+              <br />
+            </div>
+          </div>
+        </div>
+      <% end %>
+  </div>
+</div>
```
Comment (dothidiemthao) [inline app/views/home_pages/_list_documents.html.erb]: end line

File: app/views/home_pages/index.html.erb [hunk 5]
```diff
@@ -0,0 +1 @@
+<%= render "list_documents" %>
```
Comment (dothidiemthao) [inline app/views/home_pages/index.html.erb]: end line

**Comment by dothidiemthao** [comment]
commented

**Comment by trangkarol** [comment]
ready

**Pull request merged**